### ISSUE TITLE
unable to reproduce the results of BTCV, only a mean dice score of 0.2.

### ISSUE THREAD

**Issue by XinYue-Yu1031** [ge-xing/Diff-UNet#39]
Hello,

I trained BTCV on a single V100 GPU with a random data split ratio of 4:1, using a batch size of 1 and training for 3000 epochs. However, my mean dice score is only 0.2. Do you have any good training suggestions, or could you provide your BTCV training model? I would greatly appreciate it.
![1](https://github.com/user-attachments/assets/abd9736e-4553-4e16-9dbf-de99cec8b847)
![2](https://github.com/user-attachments/assets/4bd47027-cc01-449a-88c8-692612f8d223)
![3](https://github.com/user-attachments/assets/d2f91441-ced2-4527-91e3-b8b1942dd969)

**Comment by ge-xing** [comment]
The training loss is very strange. I think you should try the BraTS dataset and see if there are still any problems.

**Comment by Hwa-ZZZ** [comment]
Hello,can you tell me how is this dice and epoch graph given? Is the data plotted through the code again?Thank you!
